fe/fe-core/src/main/java/com/starrocks/alter/LakeTableAlterMetaJob.java

```diff
@@ -31,10 +31,10 @@
 import com.starrocks.common.Config;
 import com.starrocks.common.DdlException;
 import com.starrocks.common.FeConstants;
-import com.starrocks.common.MarkedCountDownLatch;
 import com.starrocks.common.Pair;
 import com.starrocks.common.io.Text;
 import com.starrocks.common.util.PropertyAnalyzer;
+import com.starrocks.common.util.concurrent.MarkedCountDownLatch;
```

Comment:
MarkedCountDownLatch seems not useful?

Reply:
useful, check the code, besides i dont change the code, just put all the lock impl stuff in a same package